doc/nrf/gs_installing.rst

```diff
@@ -487,30 +499,7 @@ To install the Nordic Edition, complete the following steps:
 
           * `SEGGER Embedded Studio (Nordic Edition) - Mac OS x64`_
 
-#. Extract the downloaded package in the directory of your choice.
-#. Register and activate a free license.
-   |SES| is free of charge for use with Nordic Semiconductor devices, but you still need to request and activate a license.
-   Complete the following steps:
-
-    a. Navigate to the :file:`bin` directory.
-    #. Run the :file:`emStudio` executable file.
-       SES first opens the Options window, which you can close at this stage, and then the Dashboard window, which informs you about the missing license.
-
-        .. figure:: images/ses_license.PNG
-           :alt: SEGGER Embedded Studio Dashboard notification about missing license
-
-           No commercial-use license detected SES prompt
-
-    #. Click :guilabel:`Activate Your Free License`.
-       A request form appears.
-
-    #. Fill in your information and click :guilabel:`Request License`.
-       The license is sent to you in an email.
-
-    #. After you receive your license key, click :guilabel:`Enter Activation Key` to activate the license.
-
-    #. Copy-paste the license key and click :guilabel:`Install License`.
-       The license activation window will close and SES will open the Project Explorer window.
+#. Extract the downloaded package in the directory of your choice. |SES| is free of charge for use with Nordic Semiconductor devices.
```

Comment:
We always start new sentences from a new line. See point 4 under "General documentation guidelines" here: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/doc_styleguide.html#doxygen-gl